Include/internal/pycore_runtime.h

```diff
@@ -53,12 +53,56 @@ typedef struct _Py_AuditHookEntry {
     void *userData;
 } _Py_AuditHookEntry;
 
+typedef struct _Py_DebugOffsets {
```

Comment:
I suggest adding the Python hex version as the first field of this structure, and committing to backwards compatibility for that one field.

That won't be very useful at first, since debuggers will want to support both older Python versions that don't have this field and newer ones that do, so they'll need to determine version to figure out if this structure even exists. But, 5 years down the line, when every non-EOL Python version has this structure, this will let debuggers decide which minor version's structure layout to use by looking only at the `PyRuntimeState`.

In fact... maybe it would even be worth including a cookie, so that debuggers could determine if the runtime state includes this structure starting from the very first CPython version that supports it.

Maybe make the first two fields:
```c
char cookie[8] = "xdebugpy";
uint64_t version = PY_VERSION_HEX;
```

That seems like a good cookie because it can't be mistaken for a word-aligned pointer regardless of 32- or 64-bit, and regardless of big or little endian, so no previous version's runtime state should ever have been able to start with that value (they all started with either enums, bools, or pointers).

Reply:
Done!